macros/fivetran/base/fivetran_fb_ad_creatives.sql

```diff
@@ -0,0 +1,46 @@
+{% macro fivetran_fb_ad_creatives() %}
+
+    {{ adapter_macro('facebook_ads.fivetran_fb_ad_creatives') }}
+
+{% endmacro %}
+
+
+{% macro default__fivetran_fb_ad_creatives() %}
+
+with base as (
+
+    select
+
+        id,
```

Comment:
I would rename this to `creative_id`- it will allow you to use using in downstream models

Reply:
for this and the other IDs in the macros, should I update the stitch macros as well? I left these as `id` based on the existing macros

Reply:
i think we should update for both stitch and fivetran